CONTRIBUTING.md

```diff
@@ -1348,59 +1348,59 @@ can be found [on the wiki][wiki-CUDW].
+[needs-documentation]: https://github.com/rocq-prover/rocq/labels/needs%3A%20documentation
+[needs-fixing]: https://github.com/rocq-prover/rocq/labels/needs%3A%20fixing
+[needs-rebase]: https://github.com/rocq-prover/rocq/labels/needs%3A%20rebase
+[needs-testing]: https://github.com/rocq-prover/rocq/labels/needs%3A%20testing
+[needs-test-suite]: https://github.com/rocq-prover/rocq/labels/needs%3A%20test-suite%20update
+[Nix]: https://github.com/rocq-prover/rocq/wiki/Nix
 [notification-email]: https://blog.github.com/2017-07-18-managing-large-numbers-of-github-notifications/#prioritize-the-notifications-you-receive
 [OCaml-planet]: http://ocaml.org/community/planet/
 [ocamlformat]: https://github.com/ocaml-ppx/ocamlformat
 [ocamlverse-community]: https://ocamlverse.github.io/content/community.html
 [old-style-guide]: dev/doc/style.txt
-[standard-library]: https://github.com/coq/stdlib
-[other-standard-libraries]: https://github.com/coq/stdlib2/wiki/Other-%22standard%22-libraries
+[standard-library]: https://github.com/rocq-prover/stdlib
+[other-standard-libraries]: https://github.com/rocq-prover/stdlib2/wiki/Other-%22standard%22-libraries
 [pinentry-mac]: https://stackoverflow.com/questions/39494631/gpg-failed-to-sign-the-data-fatal-failed-to-write-commit-object-git-2-10-0
-[Platform-docs]: https://github.com/coq/platform-docs
+[Platform-docs]: https://github.com/rocq-prover/platform-docs
 [plugin-tutorial]: doc/plugin_tutorial
 [Proof-Assistants-SE]: https://proofassistants.stackexchange.com/
 [ProofGeneral-issues]: https://github.com/ProofGeneral/PG/issues
 [Reddit]: https://www.reddit.com/r/Coq/
-[refman]: https://coq.inria.fr/distrib/current/refman/
+[refman]: https://rocq-prover.org/distrib/current/refman/
 [refman-sources]: doc/sphinx
 [refman-README]: doc/sphinx/README.rst
-[release-plan]: https://github.com/coq/coq/wiki/Release-Plan
+[release-plan]: https://github.com/rocq-prover/rocq/wiki/Release-Plan
 [repology-coq]: https://repology.org/project/coq/versions
 [Rocq-planet]: https://rocq-prover.org/rocq-planet
 [RM-checklist]: dev/doc/release-process.md
 [Stack-Exchange]: https://stackexchange.com/filters/299857/questions-tagged-coq-on-stackexchange-sites
-[Stack-Exchange-to-Zulip]: https://coq.zulipchat.com/#narrow/stream/237977-Coq-users/topic/New.20Stack.20Exchange.20question
+[Stack-Exchange-to-Zulip]: https://rocq-prover.zulipchat.com/#narrow/stream/237977-Rocq-users/topic/New.20Stack.20Exchange.20question
 [Stack-Overflow]: https://stackoverflow.com/questions/tagged/coq
-[stdlib-doc]: https://coq.inria.fr/stdlib/
+[stdlib-doc]: https://rocq-prover.org/stdlib/
 [test-suite-README]: test-suite/README.md
-[tools-website]: https://coq.inria.fr/related-tools.html
-[tools-wiki]: https://github.com/coq/coq/wiki/Tools
-[unreleased-changelog]: https://coq.github.io/doc/master/refman/changes.html#unreleased-changes
+[tools-website]: https://rocq-prover.org/related-tools.html
```

Comment:
This used to redirect to https://github.com/rocq-community/awesome-coq?tab=readme-ov-file#tools on the previous website. The redirect is currently broken. Will add, but perhaps it is better to redirect directly there:
```suggestion
[tools-website]: https://github.com/rocq-community/awesome-coq?tab=readme-ov-file#tools
```